examples/flutter_gallery/lib/gallery/home.dart

```diff
@@ -52,37 +52,40 @@ class _CategoryItem extends StatelessWidget {
     final ThemeData theme = Theme.of(context);
     final bool isDark = theme.brightness == Brightness.dark;
 
-    return new RawMaterialButton(
-      padding: EdgeInsets.zero,
-      splashColor: theme.primaryColor.withOpacity(0.12),
-      highlightColor: Colors.transparent,
-      onPressed: onTap,
-      child: new Column(
-        mainAxisAlignment: MainAxisAlignment.end,
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: <Widget>[
-          new Padding(
-            padding: const EdgeInsets.all(6.0),
-            child: new Icon(
-              category.icon,
-              size: 60.0,
-              color: isDark ? Colors.white : _kFlutterBlue,
-            ),
-          ),
-          const SizedBox(height: 10.0),
-          new Container(
-            height: 48.0,
-            alignment: Alignment.center,
-            child: new Text(
-              category.name,
-              textAlign: TextAlign.center,
-              style: theme.textTheme.subhead.copyWith(
-                fontFamily: 'GoogleSans',
+    // Contain the repaint from the splash.
+    return new RepaintBoundary(
```

Comment:
```
// This repaint boundary prevents the entire app from being rebuilt when 
// the _CategoriesPage is rebuilt due to the backdrop's front layer opening
// or closing.
```

Reply:
It's the other way around. Reworded a bit.